### PR TITLE
Add Install Juju button to canvas

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA
- Click the Install Juju button in the bottom right and check it opens https://jujucharms.com/docs/stable/getting-started in a new tab
- Check it matches the design [1]

[1] https://drive.google.com/a/canonical.com/file/d/0B2sqJRO7vy3ZbTRGOXpWd1lxVW8/view

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4138/
Test FAILed.

**Comment by hatched** [comment]
This will be shown even for real Juju environments - it should probably only be shown in sandbox mode.

**Comment by anthonydillon** [comment]
I double checked with Luca and he said it was to show on all environments.

**Comment by hatched** [comment]
:+1: :ok:
I still find it odd that this will be shown even in situations where the user has already installed it :)

File: jujugui/static/gui/src/app/components/deployment-bar/_deployment-bar.scss [hunk 1]
```diff
@@ -15,13 +15,21 @@
...
+        left: -170px;
     }
 
     &__file {
         @extend %deployment-bar__links;
         visibility: hidden;
     }
+
+    &__install-button {
+      @extend %floating-panel;
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/deployment-bar/_deployment-bar.scss]: Four space indent here, just to make it consistent.

**Comment by huwshimi** [comment]
Nice work @anthonydillon QA OK :+1: (although I agree with @hatched that it's strange to have an install link when you already have Juju installed).

File: jujugui/static/gui/src/app/components/deployment-bar/_deployment-bar.scss (hunk 1 above)
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/deployment-bar/_deployment-bar.scss]: Great spot, thanks.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4148/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4149/
Test PASSed.

**Comment by anthonydillon** [comment]
Thanks. Will land, as it was asked and will ask @Lucapaulina again. :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**